Gemfile

```diff
@@ -49,7 +49,7 @@ gem 'money-rails', '~> 1.12'
 
 gem 'ransack', '~> 2.3'
 
-gem 'sidekiq', "< 7"
+gem 'sidekiq', "6.5.12"
```

Comment:
use the sperm operator pls `~> 6.5.12`

Reply:
Added on 781502fd604597e1e638945663cf38c43baff5a6